consumers/notifiers/service.go

```diff
@@ -127,3 +127,35 @@ func (ns *notifierService) Consume(message interface{}) error {
 
 	return nil
 }
+
+func (ns *notifierService) ConsumeAsync(message interface{}, errs chan<- error) {
```

Comment:
`ConsumerAsync` makes a lot of sense in notifiers, but this does not implement the correct interface. :)
Please add checks `var _ consumers.AsyncConsumer = (*notifierService)(nil)`.